content/influxdb/v2.0/write-data/developer-tools/csv.md

```diff
@@ -586,8 +581,19 @@ csv.from(url: "https://influx-testdata.s3.amazonaws.com/noaa.csv")
 ```
 
 {{% note %}}
-#### Required columns
-To write data to InfluxDB with Flux, data must include the following columns:
+#### Required annotations and columns
+To write CSV data to InfluxDB with Flux, _all_ of the following annotations and columns must be present.
+Required annotations:
+
+- `datatype`
+- `group`
+- `default`
+
+See [annotations](/influxdb/v2.0/reference/syntax/annotated-csv/#annotations) for more information.
+With Flux, there must also be a comma between the annotation name and the annotation values (this differs from the `influx write` command).
```

Comment:
With Flux, you must also include a comma between the annotation name and the annotation values (this differs from the `influx write` command).